docs/how-to/ccv0.sh

```diff
@@ -51,6 +51,10 @@ export CONTAINER_ID="${CONTAINER_ID:-0123456789}"
 source /etc/os-release || source /usr/lib/os-release
 grep -Eq "\<fedora\>" /etc/os-release 2> /dev/null && export USE_PODMAN=true
 
+if [ -d ${tests_repo_dir} ]; then
```

Comment:
I'm not sure this checking is needed. Before some variables that holds file/directory path aren't checked. Is there anything special with `BATS_TEST_DIRNAME` that requires this checking?